src/components/Modals/LeaveFilterModal/LeaveFilterModal.scss

```diff
@@ -0,0 +1,23 @@
+.leave-filter-modal {
+    width: 44rem;
+    height: unset;
+    border-radius: 8px;
+
+    @include mobile {
+        max-width: 34rem;
+    }
```

Comment:
you could use the mixin `default-modal` but override the height and mobile width, would that work